bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+| `service.loadBalancerIP`               | loadBalancerIP if kong service type is `LoadBalancer`                                               | `nil`                                                   |
+| `ingress.enabled`                      | Enable the use of the ingress controller to access the web UI                                          | `false`                                                 |
+| `ingress.certManager`                  | Add annotations for cert-manager                                                                       | `false`                                                 |
+| `ingress.annotations`                  | Annotations for the kong Ingress                                                                    | `{}`                                                    |
+| `ingress.hosts[0].name`                | Hostname to your kong installation                                                                  | `kong.local`                                         |
+| `ingress.hosts[0].paths`               | Path within the url structure                                                                          | `["/"]`                                                 |
+| `ingress.hosts[0].tls`                 | Utilize TLS backend in ingress                                                                         | `false`                                                 |
+| `ingress.hosts[0].tlsHosts`            | Array of TLS hosts for ingress record (defaults to `ingress.hosts[0].name` if `nil`)                   | `nil`                                                   |
+| `ingress.hosts[0].servicePort`            | Which service port inside the kong service to redirect                   | `nil`                                                   |
+| `ingress.hosts[0].tlsSecret`           | TLS Secret (certificates)                                                                              | `kong.local-tls`                                     |
+| `securityContext.enabled`              | Enable securityContext on for Granafa deployment                                                       | `true`                                                  |
+| `securityContext.runAsUser`            | User for the security context                                                                          | `1001`                                                  |
+| `securityContext.fsGroup`              | Group to configure permissions for volumes                                                             | `1001`                                                  |
+| `securityContext.runAsNonRoot`         | Run containers as non-root users                                                                       | `true`                                                  |
```

Comment:
Please divide this into two parameters: `podSecurityContext` and `containerSecurityContext`:
```
| `podSecurityContext` | Kong pods' Security Context | `{}` |
| `containerSecurityContext`  | Kong containers' Security Context | `{}` |
```
Also, remove `xxx.enabled` and treat them as a YAML object. It's more flexible to add other fields

Reply:
Done